projects/Amlogic-ce/packages/linux-drivers/amlogic/RTL8822CU-aml/package.mk

```diff
@@ -0,0 +1,39 @@
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2017-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="RTL8822CU-aml"
+PKG_VERSION="f1a4a3abb993245bf28bf1e2254f47f76972fd6d"
+PKG_SHA256="2ad96072107478964b9308c19fa0edefdbb70e4c088256432b6fe5b903ebcf35"
+PKG_ARCH="arm aarch64"
+PKG_LICENSE="GPL"
+PKG_SITE="https://github.com/khadas/android_vendor_wifi_driver_realtek_8822cu/"
+PKG_URL="https://github.com/khadas/android_vendor_wifi_driver_realtek_8822cu//archive/$PKG_VERSION.tar.gz"
```

Comment:
Please use current used formatting like `${PKG_VERSION}` as example.
Also there are two `//` in the `PKG_URL`.